frontend/public/components/events.jsx

```diff
@@ -415,7 +415,7 @@ class EventStream extends SafetyFirst {
           </div>
         </div>
         { count > 0 &&
-            <WindowScroller>
+            <WindowScroller scrollElement={document.getElementById('content-scrollable')}>
```

Comment:
Wonder if there's a simple way to make `document.getElementById('content-scrollable')` a global?

Reply:
I'm hesitant to hold on to the actual element reference. Not sure if it ever could be removed from the DOM. It seems safer to use `getElementById` when we need it.